tgui/src/interfaces/airlock_electronics.ract

```diff
@@ -20,6 +20,12 @@
 			{{/each}}</tr>
 		</tbody>
 	</table>
+	<hr/>
+	<span class='highlight bold'>Unrestricted Access:</span>
```

Comment:
https://github.com/tgstation/tgstation/pull/39147 This would be really good to port. @deathride58